README.md

```diff
@@ -408,7 +408,7 @@ ui-image    image   url: image-registry.openshift-image-registry.svc:5000/pipeli
 A `PipelineRun` is how you can start a pipeline and tie it to the git and image resources that should be used for this specific invocation. You can start the pipeline using `tkn`:
 
 ```bash
-$ tkn pipeline start build-and-deploy
+$ tkn pipeline start build-and-deploy --serviceaccount pipeline
```

Comment:
This is not required if you are using openshift-pipelines-operator

Reply:
Hm, if you say so. I've updated this PullRequest to no longer have the `--serviceaccount` flag.
However, I did install using the Operator, and no ServiceAccount was added to my PipelineRun (sorry for the long block of text):
```YAML
apiVersion: tekton.dev/v1alpha1
kind: PipelineRun
metadata:
  creationTimestamp: "2019-12-17T18:39:50Z"
  generateName: build-and-deploy-run-
  generation: 1
  labels:
    tekton.dev/pipeline: build-and-deploy
  name: build-and-deploy-run-gfwqv
  namespace: pipelines-tutorial
  resourceVersion: "15259860"
  selfLink: /apis/tekton.dev/v1alpha1/namespaces/pipelines-tutorial/pipelineruns/build-and-deploy-run-gfwqv
  uid: 9bf552d1-20fc-11ea-b6d5-00000a1008bb
spec:
  pipelineRef:
    name: build-and-deploy
  podTemplate: {}
  resources:
  - name: api-repo
    resourceRef:
      name: api-repo
  - name: api-image
    resourceRef:
      name: api-image
  - name: ui-repo
    resourceRef:
      name: ui-repo
  - name: ui-image
    resourceRef:
      name: ui-image
status:
  completionTime: "2019-12-17T18:41:50Z"
  conditions:
  - lastTransitionTime: "2019-12-17T18:41:14Z"
    message: TaskRun build-and-deploy-run-gfwqv-build-api-znqsk has failed
    reason: Failed
    status: "False"
    type: Succeeded
  startTime: "2019-12-17T18:39:50Z"
...
```

Reply:
Let me take a look at this

Reply:
I just tried this again and it is working for me, serviceAccountName is getting added to the pipelinerun.

I am using the openshift-pipelines-operator 0.8.2